### PR TITLE
test-configs-cip: update isar-cip-core rootfs

### PR DESCRIPTION
**Pull request by aliceinwire**
Signed-off-by: Alice Ferrazzi <alice.ferrazzi@miraclelinux.com>

### CODE REVIEW

**Comment by aliceinwire** [comment]
@nuclearcat can you check and merge this? thanks

**Comment by gctucker** [comment]
Yes, latest is this one: https://storage.kernelci.org/images/rootfs/cip/20230321/

**Comment by gctucker** [comment]
@aliceinwire Could you please update this PR with the latest CIP rootfs images?

File: config/core/build-configs-cip.yaml [hunk 1]
```diff
@@ -237,6 +237,40 @@ build_configs:
               - 'allnoconfig'
               - 'cip://5.10.y-cip/riscv/qemu_riscv64_defconfig'
 
+  cip_6.1:
```
Comment (patersonc) [inline config/core/build-configs-cip.yaml:240]: Hello @aliceinwire 

I believe we need to move this below `cip_5.10-rt` to maintain alpha order.

Thanks, Chris

Comment (aliceinwire) [reply config/core/build-configs-cip.yaml:240]: this is just the pr for updating the isar-cip-core not 6.1. I used same branch for another pr and the commit come also here. by the way for replaying to your question, there is no alpha order, the order is checked (I think alphabetically) by githubCI and that's the only order that doesn't give an error on the PR

**Pull request closed without merging**